### PR TITLE
chore(ci): switch to upstream for ISO generation

### PR DESCRIPTION
**Pull request by noelmiller**
# Purpose

Jason has put a ton of work into his iso generation project we use upstream. I would like to switch to using it so we don't need to maintain a fork moving forward. I will add flatpak support in a separate PR.

### CODE REVIEW

File: .github/workflows/build_iso.yml [hunk 1]
```diff
@@ -39,24 +39,27 @@ jobs:
             image_tag: gts
     steps:
       - name: Build ISOs
-        uses: ublue-os/isogenerator@1.0.9
+        uses: jasonn3/build-container-installer@main
```
Comment (castrojo) [inline .github/workflows/build_iso.yml:42]: I'd like to pin a version here instead of main. (We'll get our updates via dependabot)

**Review by castrojo — CHANGES_REQUESTED** [review]

File: .github/workflows/build_iso.yml (hunk 1 above)
Comment (noelmiller) [reply .github/workflows/build_iso.yml:37]: main is being used for testing for now until he publishes a release.

Comment (castrojo) [reply .github/workflows/build_iso.yml:37]: Gotcha!

**Review by castrojo — DISMISSED** [review]

**Comment by noelmiller** [comment]
For 38 GTS images, bootc switch command is not working. This is likely due to version of bootc that is shipped with gts. 
image

**Comment by noelmiller** [comment]
Closing PR, moving to testing

**Pull request closed without merging**